topics/07_web_ui_testing/hw_ui_test.py

```diff
@@ -0,0 +1,46 @@
+import pytest

```

Comment:
Where is the PO requested by the homework? While the tests are functioning as intended, they are not following the PO standard as requested in the task description. The execution steps of the automation are all within the test function itself, and not divided into page objects and test methods as suggested by the PO model.